### ISSUE TITLE
bugs in DFNet.py

### ISSUE THREAD

**Issue by cxt98** [Galaxies99/TransCG#5]
First one is nn.BatchNorm2d(), which should only be called with one #num_features parameter. Now in most of its calls in __init__ function, it's called like 'nn.BatchNorm2d(self.hidden_channels, self.hidden_channels)' which will assign param 'eps' to #num_features.
Second one is interpolation of depth image, which should follow the 'nearest' mode instead of 'bilinear', because depth pixels could be incontinuous and linear interpolation will create non-existing points, can use 'torchvision.transforms.Resize((H//2, W//2))(depth)' instead.

The code still works well on depth completion with the bugs though. Thanks for contributing the source code!
